### PR TITLE
IronRDP addition

### PR DESCRIPTION
**Pull request by ibeckermayer**
Adds information about IronRDP.

I'm planning to study how IronRDP connects to devolutions-gateway and how it connects to the RDP server itself in order to be able to speak more intelligently about the two options.

### CODE REVIEW

File: rfd/0107-desktop-access-performance-research.md [hunk 1]
```diff
@@ -91,12 +89,121 @@ sequenceDiagram
...
++----------------------+     +------------------+  +------------------+
+|                      |     |                  |  |                  |
+|  (IronRDP)           |     |                  |  |                  |
+|  User's Web Browser  -------  Devolutions     ----  Windows Desktop |
+|                      |     |  Gateway         |  |                  |
++----------------------+     +------------------+  +------------------+
+```
+
+Given that `IronRDP` and `rdp-rs` are both RDP clients written in Rust, the most obvious option is to simply swap one out for the other.
+As ever, this is easier said than done for a variety of reasons including primarily that `IronRDP` is written to connect specifically to [Devolutions Gateway](https://github.com/Devolutions/devolutions-gateway),
```
Comment (LKozlowski) [inline rfd/0107-desktop-access-performance-research.md:184]: Is it always necessary to use the gateway? They implemented a client in the tauri gui that doesn't seem to use gateway: https://github.com/Devolutions/IronRDP/blob/master/iron-tauri-client/src-tauri/src/main.rs#L228 I might be wrong about this as I did not fully check how everything works under the hood in the IronRDP

Comment (ibeckermayer) [reply rfd/0107-desktop-access-performance-research.md:183]: Ah interesting point, maybe you're right. I'll take a closer look.

**Comment by LKozlowski** [comment]
Do you think that comparing our current implementation and our addition to the `rdp-rs` library with the features of IronRDP and making a list of missing features would be a good fit for this PR as well?

**Review by LKozlowski — APPROVED** [review]

File: rfd/0107-desktop-access-performance-research.md [hunk 2]
```diff
@@ -90,10 +88,166 @@ sequenceDiagram
 - proxy forwards message to web browser
 - web browser loads bitmap from the in-memory cache
 
+#### session recordings
```
Comment (zmb3) [inline rfd/0107-desktop-access-performance-research.md:91]: I think this has since been removed.

**Review by zmb3 — APPROVED** [review]

**Pull request merged**